<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuMain</class>
 <widget class="QWidget" name="MenuMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>91</width>
    <height>108</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Open Sans</family>
    <pointsize>9</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QListView {
    show-decoration-selected: 1; /* make the selection span the entire width of the view */
	outline: none;
}

QListView::item:alternate {
    background: #EEEEEE;
	outline: none;
}

QListView::item:selected {
    background: #2980b9;
	border-radius: 3px;
	outline: none;
}

QListView::item:selected:!active {
    /*background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #ABAFE5, stop: 1 #8588B2);*/
	outline: none;
}

QListView::item:selected:active {
    /*background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #6a6ea9, stop: 1 #888dd9);*/
	outline: none;
}

QListView::item:hover {
    background: #34495e;
	color: #2980b9;
	border-radius: 3px;
	outline: none;
}

QListView::item {
	height: 21px;
	color: #ecf0f1;
}</string>
  </property>
  <widget class="QLabel" name="label_arrow_left">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>0</y>
     <width>16</width>
     <height>8</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../Design/design.qrc">:/Menu/arrow</pixmap>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>8</y>
     <width>91</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2C3E50;
border-radius: 3px;
border-top-left-radius: 0px;
border-top-right-radius: 0px;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideRight</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="movement">
    <enum>QListView::Snap</enum>
   </property>
   <property name="flow">
    <enum>QListView::TopToBottom</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>false</bool>
   </property>
   <property name="resizeMode">
    <enum>QListView::Adjust</enum>
   </property>
   <property name="layoutMode">
    <enum>QListView::SinglePass</enum>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
   <property name="uniformItemSizes">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="selectionRectVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_arrow_right">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>16</width>
     <height>8</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../Design/design.qrc">:/Menu/arrow</pixmap>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../Design/design.qrc"/>
 </resources>
 <connections/>
</ui>
